emspy/query/fltquery.py

```diff
@@ -1,627 +1,845 @@
-from __future__ import print_function
 from __future__ import absolute_import
-from future.utils import string_types
-from builtins import zip
+from __future__ import print_function
+
+import json
+import sys
 from builtins import str
+from builtins import zip
 from collections import OrderedDict
-from emspy.query import *
-from .query import Query
 
 import pandas as pd
-import sys, json
```

Comment:
There's a lot to go through here with the PEP8 changes and whatever else was done. Can you summarize the non-PEP8 changes so I know where to look?

Reply:
- Added the `deselect` method to drop fields from a query
- Modified `filter` with an `operator` parameter to control if the filters are joined with AND or OR
- Added `remove_filter` method to drop a filter from the query
- Made `__split_expr` method a static method
- Modified `__split_expr` to account for `is null` and `is not null` operators
- Added support for `is null` and `is not null` operators, added these as keys to the `sp_ops` dictionary
- Modified each filter function to account for `is null` and `is not null` operators